src/SymfonyRequestMessage.php

```diff
@@ -2,22 +2,37 @@
 
 namespace HttpSignatures;
 
+use Symfony\Component\HttpFoundation\ParameterBag;
+use Symfony\Component\HttpFoundation\Request;
```

Comment:
It's how I've tended to do it and seen it done: http://www.phpdoc.org/docs/latest/guides/types.html#class-name